sim/firesim-lib/src/main/cc/fesvr/firesim_dtm.cc

```diff
@@ -0,0 +1,113 @@
+// See LICENSE for license details
+#include "firesim_dtm.h"
+#include <inttypes.h>
+#include <stdio.h>
+
+#define fprintf(stdout, fmt, ...) (0)
+
+firesim_dtm_t::firesim_dtm_t(int argc, char **argv, bool can_have_loadmem)
+    : testchip_dtm_t(argc, argv, can_have_loadmem), is_busy(false),
+      is_loaded_in_host(false), is_loaded_in_target(false) {
+  idle_counts = 10;
```

Comment:
@abejgonzalez Can you remind me how this number was decided?

Reply:
The `firesim_dtm_t` class is mainly a copy of the corresponding `*tsi_t` version. So that member is a copy of the same member of that class. It is a legacy member that was put in the original FireSim release. From what I can see, it's just used for the blockdevice bridge to save on MMIO calls if not needed.

Reply:
Oh I meant why idle_count is set to 10, not some other number?

Reply:
That I don't know. I think its arbitrary